lib/image-stream/README.md

```diff
@@ -58,4 +58,11 @@ These are the rules for handling archive images:
 
 The module throws an error if the above rules are not met.
 
+Supported Formats
+-----------------
+
+There are currently three image types in supported formats: `image`, `compressed` and `archive`.
+Note that when marking an extension as `compressed`, the filename will be stripped of that extension,
+and the leftover extension examined to determine the compressed image format (i.e. `.img.gz -> .img`).
```

Comment:
I guess this is a potential pitfall of identifying mime-types purely by file extension ;-)